controls/radgridview/export/how-to/export-datetime.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Export DateTime Value
+page_title: Export DateTime Value
+description: Export DateTime Value
+slug: gridview-export-datetime
+tags: export, datetime, value
+published: True
+position: 5
+---
+
+# Export DateTime Value
+
+__PROBLEM__
+
+1) When exporting DateTime values without specified **DataFormatString** to Excel, the values appear as numbers as shown in **Figure 1**:
+
+#### __Figure 1: Exporting DateTime values without specified DataFormatString__
+
+![Exporting DateTime values without specified DataFormatString](images/gridview-export-datetime-as-number.png)
+
+2) When exporting DateTime values with specified **DataFormatString** to Excel, the values appear as strings as shown in **Figure 2**:
+
+#### __Figure 2: Exporting DateTime values with specified DataFormatString__
+
+![Exporting DateTime values with specified DataFormatString](images/gridview-export-datetime-as-string.png)
+
+__CAUSE__
+
+1) In most modern programming environments, dates are stored as real numbers. The integer part of the number is the number of days since some agreed-upon date in the past, called the epoch. In Excel, June 16, 2006, for example, is stored as 38884, counting days where January 1st, 1900 is 1.
+
+2) When a DataFormatString has been specified for a given column, RadGridView exports the string representation of the values in that column.
+
+__SOLUTION__
+
+When an element is exported through the __ExportToXlsx__ or __ExportToPdf__ methods, the arguments of the __ElementExportingToDocument__ event can be cast to __GridViewCellExportingEventArgs__. Using their **VisualParameters** property and casting that to **GridViewDocumentVisualExportParameters**, we can set the **CellValueFormat** for the **CellSelectionStyle** in order to specify how the values should be formatted in Excel.
```

Comment:
I would make that a more general explanation of the solution. Something like " When an element is exported through the __ExportToXlsx__ or __ExportToPdf__ methods, the arguments of the __ElementExportingToDocument__ event  can be used to modify the visual appearance of the exported values. This is achieved through the X property..." I don't see a point at specifying something needs to be cast to an object of a specific type.